frontEnd/tickets-app/src/components/Film/Film.js

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+import "./Film.scss";
+
+const Film = ({ img, dateEnd, dateStart, name }) => {
+  const start = useState(new Date(dateStart));
```

Comment:
you should always use it like `const [start, setStart] = useState(new Date())`. use always a destructure for `useState`